### PR TITLE
Syndicate Target Practice fixes

### PR DESCRIPTION
**Pull request by Pointedstick**
Made the syndicate target ships into self-destructing automata, imposed a penalty for failure, and added some appropriate hostile and disabled hails, thereby resolving the following issues:
- https://github.com/endless-sky/endless-sky/issues/2151
- https://github.com/endless-sky/endless-sky/issues/2268
- https://github.com/endless-sky/endless-sky/issues/2890

I also wanted add a fine on failure, but that currently doesn't show up right in the description. I have opened issue https://github.com/endless-sky/endless-sky/issues/2900 to track that.

### CODE REVIEW

File: data/hails.txt [hunk 1]
```diff
@@ -3041,7 +3041,7 @@ phrase "Test Dummy"
 		"Warning! Warning!"
 		"Warning! Warning! Alien spacecraft approaching!"
 
-phrase "disabled Test Dummy
+phrase "disabled Test Dummy"
 	word
 		"I'll be back."
 		"All those times... I said 'kill all humans'... I'd always whisper 'except one. You were that one."
```
Comment (Bladewood) [inline data/hails.txt]: on a note, should there be another apostrophe on this line after "whisper"?

Comment (tehhowch) [inline data/hails.txt]: you mean, after the `one` in "'except one."

Comment (Pointedstick) [inline data/hails.txt]: Fixed!

**Comment by Pointedstick** [comment]
Whoops! Gotta get that Bender quote right.

**Comment by Bladewood** [comment]
wait lol I forgot
`		"All those times... I said 'kill all humans'... I'd always whisper 'except one.' You were that one."`
The inner quotes should have double quotation marks. The outer quotes should be backticks.

File: data/hails.txt [hunk 2]
```diff
@@ -3022,6 +3022,38 @@ phrase "hostile free worlds"
...
+		"Life is about decisions. Make the wrong ones and you'll wind up face down in a pool of your own blood and urine."
+		"Hey baby, wanna kill all humans?"
+		"Danger! Danger!"
+		"Warning! Warning!"
+		"Warning! Warning! Alien spacecraft approaching!"
+
+phrase "disabled Test Dummy"
+	word
+		"I'll be back."
+		"All those times... I said 'kill all humans'... I'd always whisper 'except one.' You were that one."
```
Comment (Amazinite) [inline data/hails.txt]: Feels like some "I have no mouth and I must scream" level stuff.

File: data/syndicate world jobs.txt [hunk 3]
```diff
@@ -195,14 +220,40 @@ mission "Syndicate target practice [1]"
 		random < 20
 		"combat rating" > 50
 		"combat rating" < 200
+		"destroyed Syndicate target ship" < 1
```
Comment (marphod) [inline data/syndicate world jobs.txt]: Destroy one ship and never again seems a little harsh. Accidents do happen.  Three would seem more appropriate.

File: data/hails.txt [hunk 4]
```diff
@@ -3022,6 +3022,38 @@ phrase "hostile free worlds"
...
+		"I'll be back."
+		`All those times... I said "kill all humans"... I'd always whisper "except one." You were that one.`
+		"This is the worst kind of discrimination there is: the kind against me!"
+		"The chances of survival are 725 to 1!"
+		"We're doomed."
+		"I'm quite beside myself."
+		"I can assure you they will never get me onto one of those dreadful starships."
+		"Curse my metal body, I wasn't fast enough! It's all my fault!"
+		"I don't know what all this trouble is about, but I'm sure it must be your fault."
+
```
Comment (marphod) [inline data/hails.txt]: Nothing from Marvin the Paranoid Android or Eddie the shipboard computer (Hitchhiker's Guide to the Galaxy)?  I'm disappointed.

Comment (Pointedstick) [inline data/hails.txt]: Believe it or not, I've actually never read Hitchhiker's Guide to the Galaxy. Shoot some appropriate quotes my way in a comment here and I'll see what I can do.

Comment (Bladewood) [inline data/hails.txt]: oh god I'm not alone

Comment (gunqqer) [inline data/hails.txt]: You heathens haven't read Hitchhiker's Guide to the Galaxy? Jeesh, uncultured swine. Hearing this make me hurt, like all the diode down my left side are broken, and never will be replaced...

File: data/syndicate world jobs.txt [hunk 5]
```diff
@@ -323,9 +326,9 @@ mission "Syndicate target practice [3]"
 		distance 1 3
 		government Syndicate
 	to offer
-		random < 20
+		random < 100
```
Comment (tehhowch) [inline data/syndicate world jobs.txt]: Why even include the condition?

(unless you `oops'd` and forgot to drop it back after testing :innocent: )

File: data/hails.txt (hunk 4 above)
Comment (marphod) [inline data/hails.txt]: - I have a pain in all the diodes down my left side. (Marvin)
- Share and Enjoy. (Motto of the marketing division Sirus Cybernetics Corp.)
- Go stick your head in a pig. (What the motto means in the local language after the 100-foot-high letters sank halfway into the mud)
- We have normality. I repeat, we have normality. Anything you still can't cope with is therefore your own problem. (Eddie the shipboard computer)
- So long, and thanks for all the fish. (dolphins final message to humanity)
- Oh no, not, again. (a bowl of petunias as it plummeted out of the sky)
- A common mistake that people make when trying to design something completely foolproof is to underestimate the ingenuity of complete fools. (I  forget the context, alas.)
- For a moment, nothing happened. Then, after a second or so, nothing continued to happen. (again, forget context)
- The first million years, they were the worst.  Then the second million years, they were the worst, too.  The third million years were the worst.  Then I went into a bit of a decline (Marvin)
- Belgium.  Belgium, man. Belgium. (Belgium, of course, being the rudest word in  existence.)

there are many, many more.

File: data/syndicate world jobs.txt (hunk 5 above)
Comment (Pointedstick) [inline data/syndicate world jobs.txt]: Aargh, oops!

File: data/hails.txt (hunk 4 above)
Comment (Pointedstick) [inline data/hails.txt]: Thanks!

File: data/governments.txt [hunk 6]
```diff
@@ -340,6 +340,8 @@ government "Syndicate (Extremist)"
 government "Test Dummy"
 	swizzle 3
 	"player reputation" -1000
+	"hostile hail" "Test Dummy"
```
Comment (Amazinite) [inline data/governments.txt]: Lowercase the new hail names for the sake of consistency?

Comment (Pointedstick) [inline data/governments.txt]: Fair enough!

**Comment by tehhowch** [comment]
If #2959 merges, you can simplify the ship definitions :+1:

**Comment by tehhowch** [comment]
Due to the peculiarities of how NPC Load and Instantiate work, if you are passing in child attributes to a `ship` node, one of them needs to be `name <ship name>` or you get a nameless variant spawned.
But, you still have to have a 3rd entry on the `ship` node line, otherwise Ship::Load thinks it's a full ship designation rather than an addendum, and thus doesn't give it any outfits, sprites, etc etc. Which understandably doesn't work out very well :laughing: 
_______
PR as-is
image
_______
With "name" node
image

**Comment by Pointedstick** [comment]
You're a wizard, @tehhowch

**Comment by MessyMix** [comment]
Congratulations! This is awesome. I love every bit of it! 🎉

**Pull request merged**